localstripe/resources.py

```diff
@@ -2019,6 +2019,110 @@ def statement_descriptor(self):  # Support Stripe API <= 2018-02-05
         return Product._api_retrieve(self.product).statement_descriptor
 
 
+class Payout(StripeObject):
+    object = 'payout'
+    _id_prefix = 'po_'
+
+    def __init__(self, amount=None, currency=None, description=None,
+                 metadata=None, statement_descriptor=None, destination=None,
+                 method=None, source_type=None, status=None, **kwargs):
+        if kwargs:
+            raise UserError(400, 'Unexpected ' + ', '.join(kwargs.keys()))
+
+        amount = try_convert_to_int(amount)
+
+        try:
+            assert type(amount) is int and amount > 0
+            assert type(currency) is str and currency in ('eur')
```

Comment:
Beware, in Python, `('eur')` *is* the string `'eur'`. If you want to declare a list, use `('eur', )`.

However, is there a need to enforce only some currencies?